server/game/cards/07-WotW/RingOfBinding.js

```diff
@@ -0,0 +1,19 @@
+const DrawCard = require('../../drawcard.js');
+const { Phases } = require('../../Constants');
+const AbilityDsl = require('../../abilitydsl');
+
+class RingOfBinding extends DrawCard {
+    setupCardAbilities() {
+        this.persistentEffect({
+            match: (context) => context.game.currentPhase === Phases.Fate && context.game.getPlayersInFirstPlayerOrder()[0] === context.player,
```

Comment:
I think this should be a `condition:` not a `match:` Also `context.player.firstPlayer` is a more elegant way of checking if context.player is the first player :P

Reply:
I made the changes you suggested but the first two tests are still failing. do you have any ideas what the bug might be.